presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/AgentServer.java

```diff
@@ -32,6 +33,7 @@ public static void main(String[] args)
                 .setURI(URI)
                 .setPort(PORT)
                 .registerComponent(ConfigAPI.class)
```

Comment:
They have the exact same result. I prefer the separate calls because then to change the included classes, any of the classes' lines can be removed, without worrying about the trailing parenthesis or commas.